index.html

```diff
@@ -59,7 +59,151 @@ <h1>Hey there. I'm Osugo. <br> <span>I'm a software developer</span></h1>
 
 
     </div>
-    </div>
-</body>
 
-</html>
\ No newline at end of file
+
+    <section class="recent-works">
+
+
+        <div class="topper">
+            <h1>My Recent Works</h1>
+        </div>
+
+        <div class="indicator"></div>
+
+        <div class="recent-works-cards">
+            <div class="works work1">
+                <div class="work-details">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+                    <ul class="language-tags">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <a href="#" class="works-button">See Project</a>
+                </div>
+            </div>
+
+            <div class="works work2">
+                <div class="work-details">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+                    <ul class="language-tags">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <a href="#" class="works-button">See Project</a>
+                </div>
+            </div>
+
+            <div class="works work3">
+                <div class="work-details">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+                    <ul class="language-tags">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <a href="#" class="works-button">See Project</a>
+                </div>
+            </div>
+
+            <div class="works work4">
+                <div class="work-details">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+                    <ul class="language-tags">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <a href="#" class="works-button">See Project</a>
+                </div>
+            </div>
+
+            <div class="works work5">
+                <div class="work-details">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+                    <ul class="language-tags">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <a href="#" class="works-button">See Project</a>
+                </div>
+            </div>
+
+            <div class="works work6">
+                <div class="work-details">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
```

Comment:
- [ ] Kindly I recommend you to use header tag `hx(h3)` tag for the project title (e.g., "Multi-Post Stories Gain+Glory") instead of `h1` tag.